target-platform/target-platform.target

```diff
@@ -5,12 +5,14 @@
 	<locations>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 			<repository location="https://download.eclipse.org/eclipse/updates/4.28/"/><!--Keep in sync with repo-ref in org.eclipse.m2e.repository/category.xml-->
+			<repository location="https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2023-09/"/>
```

Comment:
Even though from the XSD it is supported this it is not really supported by the PDE UI so I think we better use a separate location here.

Reply:
Yes the UI does not handle that so nice. But that's effectivly how those IU-Locations are resolved.
Any way, I changed it as you suggested.